### PR TITLE
feat(setting): lazy update for volume-related settings

### PR DESCRIPTION
**Pull request by mantissahz**
Modify setting test cases for the new behavior.

Ref: longhorn/longhorn#7173

### CODE REVIEW

File: manager/integration/tests/test_settings.py [hunk 1]
```diff
@@ -358,6 +354,8 @@ def check_tolerations_set(current_toleration_list, expected_tolerations,
         for removed in chk_removed_tolerations:
             if current_toleration == removed:
                 unexpected += 1
+    print("[James_DBG] len(expected_tolerations): {}, found: {}, \
```
Comment (innobead) [inline manager/integration/tests/test_settings.py:357]: Remove the personal log.

**Review by innobead — CHANGES_REQUESTED** [review]
Please also update the test description comment as well, if the behavior has been changed.

File: manager/integration/tests/test_settings.py (hunk 1 above)
Comment (mantissahz) [reply manager/integration/tests/test_settings.py]: Removed. Thanks.

**Review by yangchiu — APPROVED** [review]
LGTM

**Review by innobead — APPROVED** [review]

**Comment by yangchiu** [comment]
@Mergify backport v1.6.x

**Pull request merged**